iree/compiler/Dialect/HAL/IR/HALTypes.cpp

```diff
@@ -297,16 +297,19 @@ enum class NumericalType : uint32_t {
   kIntegerUnsigned = 0x02,
   // TODO(benvanik): specialize with semantics from APFloat.
   kFloatIEEE = 0x03,
+  kFloatBrain = 0x04,
```

Comment:
Wait I thought bf16 stood for "big float 16". Guess not... https://en.wikipedia.org/wiki/Bfloat16_floating-point_format

Reply:
🧠